rsl_rl/networks/mlp.py

```diff
@@ -72,7 +72,7 @@ def __init__(
             total_out_dim = reduce(lambda x, y: x * y, output_dim)
             # add a layer to reshape the output to the desired shape
             layers.append(nn.Linear(hidden_dims_processed[-1], total_out_dim))
-            layers.append(nn.Unflatten(output_dim))
+            layers.append(nn.Unflatten(-1, output_dim))
```

Comment:
I think the output dim should be changed according to the right shape. Otherwise this is always adding an extra dimension.